doc/index.rst

```diff
@@ -278,6 +278,11 @@ Special typing primitives
 
       Support for the ``__orig_bases__`` attribute was added.
 
+   .. versionchanged:: 4.7.0
+
+      The implementation of ``TypedDict`` now follows that in Python 3.9 and higher,
+      where ``TypedDict`` is a function rather than a class.
```

Comment:
Good point. Maybe we should just say "closer" rather than "follows":

```suggestion
      ``TypedDict`` is now a function rather than a class.
      This brings the ``typing_extensions.TypedDict`` closer to the implementation
      of :py:mod:`typing.TypedDict` on Python 3.9 and higher.
```

Reply:
Looks like you might have accidentally applied the first suggestion here rather than the second suggestion? :)

Reply:
Indeed, thanks!